tools/delete_candidate_info.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+/**
+ * This script deletes the specified candidate information.
+ *
+ * Delete all table rows for a given candidate
+ * "Usage 1: php delete_candidate_info.php delete_candidate DCCID PSCID [confirm]";
+ * echo "Example: php delete_candidate_info.php delete_candidate 965327 dcc0007";
+ * echo "Example: php delete_candidate_info.php delete_candidate 965327 dcc0007 confirm";
+ *
+ * Delete all timepoint rows for a given candidate
+ * echo "Usage 2: php delete_candidate_info.php delete_timepoint DCCID PSCID SessionID [confirm]";
+ * echo "Example: php delete_candidate_info.php delete_timepoint 965327 dcc0007 482";
+ * echo "Example: php delete_candidate_info.php delete_timepoint 965327 dcc0007 482 confirm";
+ *
+ * Nullify instrument fields for a given instrument
+ * echo "Usage 3: php delete_candidate_info.php reset_instrument DCCID PSCID CommentID [confirm]";
+ * echo "Example: php delete_candidate_info.php reset_instrument 965327 dcc0007 965327dcc0007482121239212121";
+ * echo "Example: php delete_candidate_info.php reset_instrument 965327 dcc0007 965327dcc0007482121239212121 confirm";
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Various <example@example.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once "generic_includes.php";
+require_once "Candidate.class.inc";
+require_once "Utility.class.inc";
+
```

Comment:
I think Candidate.class.inc and Utility.class.inc are autoloaded so you don't need to require them here.